### PR TITLE
Support RawSize in the DwrfFileFooter

### PR DESCRIPTION
**Pull request by arunthirupathi**
Dwrf File rawSize support is added in this PR.

Test plan - (Please fill in how you tested your changes)
Added new unit tests.
Ran cross reader validation for this change.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-orc/src/main/java/com/facebook/presto/orc/writer/TimestampColumnWriter.java [hunk 1]
```diff
@@ -161,9 +162,12 @@ public void writeBlock(Block block)
 
                 secondsStream.writeLong(seconds);
                 nanosStream.writeLong(encodedNanos);
-                nonNullValueCount++;
+                blockNonNullValueCount++;
             }
         }
+
+        nonNullValueCount += blockNonNullValueCount;
+        return (block.getPositionCount() - blockNonNullValueCount) * NULL_SIZE + blockNonNullValueCount * (Long.BYTES + Integer.BYTES);
```
Comment (sdruzkin) [inline presto-orc/src/main/java/com/facebook/presto/orc/writer/TimestampColumnWriter.java:170]: I'd extract `(Long.BYTES + Integer.BYTES)` into a constant and add a comment why we use INT size for the nanos field despite using `LongOutputStreamV1.writeLong` for both seconds and nanos.

File: presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java [hunk 2]
```diff
@@ -734,6 +736,7 @@ public void close()
         Footer footer = new Footer(
                 numberOfRows,
                 rowGroupMaxRowCount,
+                OptionalLong.of(rawSize),
```
Comment (sdruzkin) [inline presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java:739]: Please add a test for the file footer raw size.

File: presto-orc/src/main/java/com/facebook/presto/orc/writer/TimestampColumnWriter.java (hunk 1 above)
Comment (arunthirupathi) [reply presto-orc/src/main/java/com/facebook/presto/orc/writer/TimestampColumnWriter.java]: Done.

File: presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java (hunk 2 above)
Comment (arunthirupathi) [reply presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java:742]: Done. (Also added a test for Stripe raw size as well)

**Review by shixuan-fan — APPROVED** [review]
LGTM

**Pull request merged**